Gemfile

```diff
@@ -6,14 +6,17 @@ source 'https://rubygems.org'
 plugin "bundler-inject", "~> 1.1"
 require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundler-inject") rescue nil
 
-#
-# VMDB specific gems
-#
+source 'http://rubygems.manageiq.org' do
+  gem "handsoap", "=0.2.5.5",  :require => false
+  gem "rugged",   "=0.28.2.2", :require => false
+  gem "ruport",   "=1.7.0.3"
 
-gem "manageiq-gems-pending", ">0", :require => 'manageiq-gems-pending', :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+  group :ui_dependencies do
```

Comment:
Not a fan of splitting up this group

Reply:
@Fryguy is there a nicer way? I wasn't sure what the best approach was.